examples/package.json

```diff
@@ -11,5 +11,10 @@
   },
   "keywords": [],
   "author": "toeverything",
-  "license": "MPL-2.0"
+  "license": "MPL-2.0",
+  "pnpm": {
+    "overrides": {
```

Comment:
Why should we overriding this package?

Reply:
https://github.com/toeverything/blocksuite/security/dependabot/15